contract_digitized_signature/models/account_analytic_account.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2017 Tecnativa - Vicent Cubells
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+
+    _name = 'account.analytic.account'
+    _inherit = ['account.analytic.account', 'mail.thread']
```

Comment:
A refactoring made in #207 to split contract object from analytic account.

Please consider this in the migration.